src/navigation/_DevToolbarButtons.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import MenuButton from '../menu-button/_MenuButton';
+
+const propTypes = {
+  /**
+   * Current Theme
+   */
+  selectedTheme: PropTypes.string,
+
+  /**
+   * Current Locale
+   */
+  selectedLocale: PropTypes.string,
+
+  /**
+   * config for themes and locales
+   */
+  config: PropTypes.shape({
```

Comment:
Should we name `devToolsConfig` to align with the prop naming in `_ComponentToolbar.jsx`?

Reply:
Removing devtoolbarbuttons